src/authorizations/components/CustomApiTokenOverlay.tsx

```diff
@@ -0,0 +1,56 @@
+import React, {FC, useState} from 'react'
+
+// Components
+import {
+  Overlay,
+  Form,
+  Input,
+  FlexBox,
+  AlignItems,
+  FlexDirection,
+  ComponentSize,
+} from '@influxdata/clockface'
+
+interface OwnProps {
+  onClose: () => void
+}
+
+const CustomApiTokenOverlay: FC<OwnProps> = props => {
+  const handleDismiss = () => {
+    props.onClose()
+  }
+  const [description, setDescription] = useState('')
+
+  const handleInputChange = event => {
+    setDescription(event.target.value)
+  }
+
+  return (
+    <Overlay.Container maxWidth={500}>
+      <Overlay.Header
+        title="Generate a Personal Api Token"
+        onDismiss={handleDismiss}
+      />
+      <Overlay.Body>
+        <Form>
+          <FlexBox
+            alignItems={AlignItems.Center}
+            direction={FlexDirection.Column}
+            margin={ComponentSize.Large}
+          >
+            <Form.Element label="Description">
+              <Input
+                placeholder="your_token_description"
```

Comment:
```suggestion
                placeholder="Enter your token's description"
```

looks like Figma had a placeholder. @daviddkkim what do you think?

Reply:
that makes sense. i've changed it to "Describe this new token"